client/src/components/postcall/PostCallOneQuestionPoll.test.tsx

```diff
@@ -76,9 +97,38 @@ describe('PostCallOneQuestionPoll', () => {
         oneQuestionPollOptions={mockOneQuestionPollInfo}
         callId={mockCallId}
         acsUserId={mockAcsUserId}
+        meetingLink={mockMeetingLink}
       />
     );
     const textField = postCallOneQuestionPoll.find(TextField);
     expect(textField.length).toBe(1);
   });
+
+  it('should trigger window location to redirect to booking', async () => {
+    const mockOneQuestionPollInfo: OneQuestionPollOptions = {
+      title: '',
+      prompt: '',
+      pollType: 'text',
+      answerPlaceholder: '',
+      saveButtonText: ''
+    };
+    const postCallOneQuestionPoll = await mount<PostCallOneQuestionPollProps>(
+      <PostCallOneQuestionPoll
+        theme={mockTheme}
+        oneQuestionPollOptions={mockOneQuestionPollInfo}
+        callId={mockCallId}
+        acsUserId={mockAcsUserId}
+        meetingLink={mockMeetingLink}
+      />
+    );
+
+    const mockSubmitSurveyResponseUtil = jest
+      .spyOn(PostCallUtil, 'submitSurveyResponseUtil')
+      .mockImplementationOnce(async () => Promise.resolve());
```

Comment:
This is necessary since it is checking if `submitSurveyResponseUtil` gets called.

Reply:
Right but will this work too?
```
    const mockSubmitSurveyResponseUtil = jest.spyOn(PostCallUtil, 'submitSurveyResponseUtil');
```

Just wondering if we can avoid the empty function.

Reply:
if we do only spyOn, it will actually try to send the api request and then we need to mock the fetch api as well. To prevent that, I just mocked the `submitSurveyResponseUtil` in the first place

Reply:
As we discussed, I tried with `jest.fn()` instead of anonymous function and it worked so I updated the code!